courses/free-courses-en.md

```diff
@@ -208,10 +208,12 @@
 * [Android App Development for Beginners Playlist](https://www.youtube.com/playlist?list=PL6gx4Cwl9DGBsvRxJJOzG4r4k_zLKrnxl) - Bucky Roberts (thenewboston)
 * [Android App Development Tutorial for Beginners (2020) \| Complete Guides and Courses](https://www.youtube.com/playlist?list=PLwhVruPHD9rz2MwQuYSenQ0WK1IGP5rYM) - tutorialsEU
 * [Android Basics: Data Storage](https://www.udacity.com/course/android-basics-data-storage--ud845) (Udacity)
+* [Android Basics in Kotlin](https://developer.android.com/courses/android-basics-kotlin/course) - Google Developers Training team (Android)
```

Comment:
I think we'll move the kotlin courses to be a subcategory under android, but for now, put the kotlin course in kotlin.

Reply:
"This course is out of date and no longer maintained."

Reply:
remove it

Reply:
Alright, I'll remove the Kotlin course. Thanks for pointing that out. Only the Jetpack Compose courses will remain